mozilla-mobile/fenix/app/src/main/res/values/strings.xml

```diff
@@ -1179,19 +1205,33 @@
     <!-- Onboarding -->
     <!-- Text for onboarding welcome message
     The first parameter is the name of the app (e.g. Firefox Preview) -->
-    <string name="onboarding_header">Welcome to %s!</string>
+    <string moz:RemovedIn="106" name="onboarding_header" tools:ignore="UnusedResources">Welcome to %s!</string>
+    <!-- Text for onboarding welcome header. -->
+    <string name="onboarding_header_2">Welcome to a better internet</string>
+    <!-- Text for the onboarding welcome message. -->
+    <string name="onboarding_message">A browser built for people, not profits.</string>
     <!-- text for the Firefox account onboarding sign in card header. The word "Firefox" should not be translated -->
-    <string name="onboarding_account_sign_in_header_1">Sync Firefox between devices</string>
+    <string moz:removedIn="106" name="onboarding_account_sign_in_header_1" tools:ignore="UnusedResources">Sync Firefox between devices</string>
```

Comment:
We usually use placeholders in place of the app name.

Reply:
Do we need to make any changes for this? This string is actually being removed I believe.

Reply:
oh yes in this case I think it's ok